ci/pipelines/tasks/run-detekt.yml

```diff
@@ -0,0 +1,15 @@
+# prepare-env task or equivalent should be run before using this
+inputs:
+- name: source
+
+outputs:
+- name: source
```

Comment:
Does this task change anything on disk?

Reply:
No

Reply:
It's more idiomatic to remove `outputs` then - I'm not sure if it affects anything functionally.